curriculum/challenges/espanol/06-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-using-a-headless-browser-ii.md

```diff
@@ -1,38 +1,38 @@
 ---
 id: 5f8884f4c46685731aabfc41
-title: Run Functional Tests Using a Headless Browser II
+title: Ejecuta pruebas funcionales usando un navegador sin interfaz gráfica II
 challengeType: 2
 forumTopicId: 301594
 dashedName: run-functional-tests-using-a-headless-browser-ii
 ---
 
 # --description--
 
-As a reminder, this project is being built upon the following starter project on [Replit](https://replit.com/github/freeCodeCamp/boilerplate-mochachai), or cloned from [GitHub](https://github.com/freeCodeCamp/boilerplate-mochachai/).
+Como recordatorio, este proyecto está siendo construido con base en el siguiente proyecto inicial [Replit](https://replit.com/github/freeCodeCamp/boilerplate-mochachai), o clonado desde [GitHub](https://github.com/freeCodeCamp/boilerplate-mochachai/).
 
 # --instructions--
 
-Within `tests/2_functional-tests.js`, in the `'submit "surname" : "Vespucci" - write your e2e test...'` test (`// #6`), automate filling-in and submitting the form from scratch:
+Dentro `tests/2_functional-tests.js`, en `'Submit the surname "Vespucci" in the HTML form'` prueba (`// #5`), automatiza lo siguiente:
 
-1.  Fill in the form with the `surname` of `Vespucci`
-2.  Submit it pressing `'submit'` button
+1.  Rellena el formulario con el apellido `Vespucci`
+2.  Presiona el botón enviar
 
-Within the callback:
+Y dentro del callback de `pressButton`:
 
-1.  assert that status is `200`
-2.  assert that the text inside the element `span#name` is `'Amerigo'`
-3.  assert that the text inside the element `span#surname` is `'Vespucci'`
-4.  assert that the element(s) `span#dates` exist and their count is `1`
+1.  Comprueba que el status es OK `200`
+2.  Comprueba que el texto dentro del elemento `span#name` es `'Amerigo'`
+3.  Comprueba que el texto dentro del elemento `span#surname` es `'Vespucci'`
+4.  Comprueba que existen elemento(s) `span#dates` y su contador es `1`
 
-Do not forget to to remove the `assert.fail()` call.
+No olvides eliminar la llamada `assert.fail()`.
 
 # --hints--
 
-All tests should pass.
+Todas las pruebas deben pasar.
 
 ```js
 (getUserInput) =>
-  $.get(getUserInput('url') + '/_api/get-tests?type=functional&n=5').then(
+  $.get(getUserInput('url') + '/_api/get-tests?type=functional&n=6').then(
```

Comment:
Has this recently been updated? I did not think so.

Reply:
This was updated in #43065 a while ago, but it looks like the update wasn't propagated to Espanol because the file hadn't been translated yet.